samples/matter/light_switch/sample.yaml

```diff
@@ -12,6 +12,7 @@ tests:
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf7002dk_nrf5340_cpuapp
   sample.matter.light_switch.debug:
     build_only: true
+    extra_args: CONFIG_CHIP_MEMORY_PROFILING=y
```

Comment:
PLease add this config to 
sample.matter.light_switch.persistent_subscriptions:
configuration too